examples/src/Auction/Offchain.hs

```diff
@@ -75,12 +76,7 @@ txSetDeadline offerUtxo deadline = do
                 NonZero 1
               )
             ],
-        txSkelIns =
-          Set.singleton $
-            SpendsScript
-              A.auctionValidator
-              A.SetDeadline
-              offerUtxo,
+        txSkelIns = Map.singleton offerUtxo $ SpendsScript A.auctionValidator A.SetDeadline,
```

Comment:
BTW I think you can do `{-# LANGUAGE OverloadedLists #-}` and write this as just
```haskell
txSkelIns = [(offerUtxo, SpendsScript A.auctionValidator A.SetDeadline)]
```

This is a rather JFYI thing — up to you if you actually want to go this route here and in other places like this.